pkg/subnet/upgrades/upgrade_file.go

```diff
@@ -47,3 +48,22 @@ func WriteUpgradeFile(jsonBytes []byte, subnetName, upgradeFilesDir string) erro
 	ux.Logger.PrintToUser("File written successfully")
 	return nil
 }
+
+func ReadUpgradeFile(subnetName, upgradeFilesDir string) ([]byte, error) {
```

Comment:
Why don't you get the `upgradesFilesDir` directly from `app` instead of asking for it as an argument? I think it will be the same argument everywhere.

Reply:
`app` is not injected into this package so both are equivalent